examples/ImageRecon/GEOmetrics/eval.py

```diff
@@ -12,147 +12,142 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import argparse
-import json
 import numpy as np
-import os
 import torch
-from torch.autograd import Variable
-import torch.optim as optim
 from torch.utils.data import DataLoader
-import sys
 from tqdm import tqdm
 
 from utils import preprocess, pooling, get_pooling_index
 from utils import setup_meshes, split_meshes, reset_meshes
-from utils import loss_surf, loss_edge, loss_lap, collate_fn
+from utils import collate_fn
 from architectures import VGG as Encoder, G_Res_Net
 from PIL import Image
+
 import kaolin as kal 
+from kaolin.datasets import shapenet
+
 parser = argparse.ArgumentParser()
-parser.add_argument('-expid', type=str, default='Direct', help='Unique experiment identifier.')
-parser.add_argument('-device', type=str, default='cuda', help='Device to use')
-parser.add_argument('-categories', type=str,nargs='+', default=['chair'], help='list of object classes to use')
-parser.add_argument('-vis', action='store_true', help='Visualize each model while evaluating')
-parser.add_argument('-f_score', action='store_true', help='compute F-score')
-parser.add_argument('-batchsize', type=int, default=1, help='Batch size.')
+parser.add_argument('--shapenet-root', type=str, help='Root directory of the ShapeNet dataset.')
+parser.add_argument('--shapenet-rendering-root', type=str, help='Root directory of the ShapeNet Rendering dataset.')
```

Comment:
I called it `shapenet-images-root` in PR #72 we should be consistent between examples, what do you prefer ?

Reply:
Happy to go with yours - it's shorter and easier to write.